src/App.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react'
+// Cannot get code to work with axios installation:
+// import axios from 'axios'
+// Code works with hard coded axios - not sure why this works, but the other way does not:
+import axios from './axios/axios'
+import styled from 'styled-components'
+
+import './App.css';
+
+import Form from './form'
+import Team from './teams'
+
+const initialFormValues = {
+  name: '',
+  email: '',
+  role: '',
+  teamName: '',
+}
+
+function App() {
+  const [teamMem, setTeamMem] = useState([])
+
+  const [formValues, setFormValues] = useState(initialFormValues)
+
+  const updateForm = (inputName, inputValue) => {
+    setFormValues({ ...formValues, [inputName]: inputValue })
+  }
+
+  const submitForm = () => {
+    const newMem = {
+      name: formValues.name.trim(),
+      email: formValues.email.trim(),
+      role: formValues.role,
+      teamName: formValues.teamName,
+    }
+
+    if (!newMem.name || !newMem.email || !newMem.role || !newMem.teamName) return
+
+    axios
+      .post(Date.now(), newMem)
+      .then(res => {
+        setTeamMem([...teamMem, res.data])
```

Comment:
If you want to add the new member to the top of the list (I think you asked this in person) then you will switch the ...teamMem and res.data so setTeamMem([res.data, ...teamMem]) should put the new member at the top of the list